.circleci/config.yml

```diff
@@ -23,3 +63,12 @@ workflows:
   build_and_test:
     jobs:
       - test
+      - build_frontend
+      - export_frontend:
```

Comment:
Could you follow this example to add the approval step?
https://github.com/mage-ai/backend/blob/master/.circleci/config.yml#L499-L515
https://github.com/mage-ai/backend/blob/master/.circleci/config.yml#L523-L531